firedrake/mg/mesh.py

```diff
@@ -80,28 +84,53 @@ def __getitem__(self, idx):
 
 def MeshHierarchy(mesh, refinement_levels,
                   refinements_per_level=1,
+                  degree=1,
                   reorder=None,
                   distribution_parameters=None, callbacks=None,
                   mesh_builder=firedrake.Mesh):
     """Build a hierarchy of meshes by uniformly refining a coarse mesh.
 
-    :arg mesh: the coarse :func:`~.Mesh` to refine
-    :arg refinement_levels: the number of levels of refinement
-    :arg refinements_per_level: the number of refinements for each
-        level in the hierarchy.
-    :arg distribution_parameters: options controlling mesh
-        distribution, see :func:`~.Mesh` for details.  If ``None``,
-        use the same distribution parameters as were used to
-        distribute the coarse mesh, otherwise, these options override
-        the default.
-    :arg reorder: optional flag indicating whether to reorder the
-         refined meshes.
-    :arg callbacks: A 2-tuple of callbacks to call before and
+    Parameters
+    ----------
+    mesh : :py:func:`~.Mesh`
```

Comment:
```suggestion
    mesh : Mesh
```

Reply:
If I do this the CI fails saying:
 ```
Warning, treated as error:
/__w/firedrake/firedrake/firedrake/mg/mesh.py:docstring of firedrake.mg.mesh.MeshHierarchy:1:py:class reference target not found: Mesh
```

Reply:
I think you want `MeshGeometry`. `Mesh` is only a function, not a class.